app/views/declaration/commodity_details.scala.html

```diff
@@ -0,0 +1,92 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import controllers.declaration.routes
+@import controllers.util.SaveAndContinue
+@import forms.Choice.AllowedChoiceValues
+@import forms.declaration.CommodityDetails
+@import forms.declaration.CommodityDetails._
+@import models.requests.JourneyRequest
+@import uk.gov.hmrc.play.views.html._
+@import services.NationalAdditionalCode
+@import services.view.AutoCompleteItem
+@import forms.declaration.AdditionalFiscalReference
+@import models.declaration.ExportItem
+@import views.Title
+
+@this(main_template: views.html.main_template)
+
+@(
+        mode: Mode,
+        itemId: String,
+        form: Form[CommodityDetails]
+)(implicit request: JourneyRequest[_], messages: Messages)
+
+@main_template(
+    title = Title("declaration.commodityDetails.title", "supplementary.items")
+) {
+
+    @helper.form(routes.CommodityDetailsController.submitForm(mode, itemId), 'autocomplete -> "off") {
+        @helper.CSRF.formField
+
+        @components.back_link(routes.FiscalInformationController.displayPage(mode, itemId), mode)
+
+        @components.error_summary(form.errors)
+
+        @components.heading(
+            sectionHeader = messages("supplementary.items"),
+            title = messages("declaration.commodityDetails.title")
+        )
+
+        @components.fields.field_text(
+            field = form(CommodityDetails.combinedNomenclatureCodeKey),
+            label = messages("declaration.commodityDetails.combinedNomenclatureCode.header"),
+            hint = Some(messages("declaration.commodityDetails.combinedNomenclatureCode.header.hint")),
+            labelClass = Some("bold-small")
+        )
+
+        @components.fields.field_textarea(
+            field = form(CommodityDetails.descriptionOfGoodsKey),
+            label = messages("declaration.commodityDetails.description.header"),
+            hint = Some(messages("declaration.commodityDetails.description.header.hint")),
+            labelClass = Some("bold-small"),
+            inputClass = Some("form-control--full-width")
+        )
+
+        <details class="govuk-details" role="group">
+            <summary class="govuk-details__summary" role="button" aria-controls="details-content" aria-expanded="false">
+                <span class="govuk-details__summary-text">
+                @messages("declaration.commodityDetails.how.header")
+                </span>
+            </summary>
+            <div class="grey-border-left">
+                <ol class="list list-number">
+                    <li>@Html(messages("declaration.commodityDetails.how.step1"))</li>
+                    <li>@Html(messages("declaration.commodityDetails.how.step2"))</li>
```

Comment:
only first one require to not be HTML escape - rest of those are text

Reply:
I know.  Just wanted to be consistent.   I have a dream where one day a content designer can edit the message file in which case they could add links.   But I'll change if you don't like